app/controllers/car_lot_vehicles_controller.rb

```diff
@@ -1,7 +1,7 @@
 class CarLotVehiclesController < ApplicationController
 
   def index
-    cars = Vehicle.all
+    cars = Vehicle.alphabetize 
```

Comment:
right idea, except that this story wants two things: 1. when we first visit the index page the children should NOT be in alphabetical order. 2. when we click the link THEN the children are sorted.

The alphabetize method is good, but we need a check for when the user clicks the alphabetize link.